internal/runners/update/update.go

```diff
@@ -64,6 +64,10 @@ func (u *Update) Run(params *Params) error {
 		return nil
 	}
 
+	if params.Version == "" {
+		u.out.Notice(locale.Tl("updating_version", "Updating State Tool to version {{.V0}}", up.Version))
+	}
```

Comment:
I think we need to update the output on line 49 above rather than add another `Updating` message

Reply:
I thought about that, but if the update check takes some time, the user is left "hanging" and wondering what is going on until we get a version to print to the output. Or is this a non-issue?

Reply:
Never mind, you were on the right track. I've pushed an update to our messaging around updates.